src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/network/PathsSpec.groovy

```diff
@@ -45,6 +46,13 @@ class PathsSpec extends HealthCheckSpecification {
             }
         }
 
+        then: "Maximum count of paths can be changed during PCE calculations"
+        def limited_paths = northbound.getPaths(switchPair.src.dpId, switchPair.dst.dpId,
+                null, null, null, null, 1)
+        assert limited_paths.paths.size() == 1
+        assert paths.paths.size() > limited_paths.paths.size()
```

Comment:
L26 should be improved.
you need to be sure that `switchPair` has two paths at least.

Reply:
I do not think that we need exactly what you asking for in this particular case. We need to check that max_path_count var can limit response output to some number of available paths. In our case all switch pairs will have more than 1 path, so our asserts will be always correct

Reply:
agree, I forgot implementation of `getNotNeighboringSwitchPair`. 
It returns switchPair with two paths at least.

Reply:
except that it is not)

Reply:
it returns a switchpair with the shortest path being of size >2 (means that switches are not neighbours, just as method name implies)